obstacle_stop_planner_refine/src/node.cpp

```diff
@@ -20,193 +20,267 @@
+    update_parameter(parameters, acc_ns + "consider_obj_velocity", acc_param_.consider_obj_velocity);
+    update_parameter(parameters, acc_ns + "obstacle_stop_velocity_thresh", acc_param_.obstacle_stop_velocity_thresh);
+    update_parameter(parameters, acc_ns + "emergency_stop_acceleration", acc_param_.emergency_stop_acceleration);
+    update_parameter(parameters, acc_ns + "obstacle_emergency_stop_acceleration", acc_param_.obstacle_emergency_stop_acceleration);
+    update_parameter(parameters, acc_ns + "emergency_stop_idling_time", acc_param_.emergency_stop_idling_time);
+    update_parameter(parameters, acc_ns + "min_dist_stop", acc_param_.min_dist_stop);
+    update_parameter(parameters, acc_ns + "max_standard_acceleration", acc_param_.max_standard_acceleration);
+    update_parameter(parameters, acc_ns + "min_standard_acceleration", acc_param_.min_standard_acceleration);
+    update_parameter(parameters, acc_ns + "standard_idling_time", acc_param_.standard_idling_time);
+    update_parameter(parameters, acc_ns + "min_dist_standard", acc_param_.min_dist_standard);
+    update_parameter(parameters, acc_ns + "obstacle_min_standard_acceleration", acc_param_.obstacle_min_standard_acceleration);
+    update_parameter(parameters, acc_ns + "margin_rate_to_change_vel", acc_param_.margin_rate_to_change_vel);
+    update_parameter(parameters, acc_ns + "use_time_compensation_to_calc_distance", acc_param_.use_time_compensation_to_dist);
+    update_parameter(parameters, acc_ns + "lowpass_gain_of_upper_velocity", acc_param_.lowpass_gain_);
+
+    /* parameter for pid in acc */
+    update_parameter(parameters, acc_ns + "p_coefficient_positive", acc_param_.p_coeff_pos);
+    update_parameter(parameters, acc_ns + "p_coefficient_negative", acc_param_.p_coeff_neg);
+    update_parameter(parameters, acc_ns + "d_coefficient_positive", acc_param_.d_coeff_pos);
+    update_parameter(parameters, acc_ns + "d_coefficient_negative", acc_param_.d_coeff_neg);
+
+    /* parameter for speed estimation of obstacle */
+    update_parameter(parameters, acc_ns + "object_polygon_length_margin", acc_param_.object_polygon_length_margin);
+    update_parameter(parameters, acc_ns + "object_polygon_width_margin", acc_param_.object_polygon_width_margin);
+    update_parameter(parameters, acc_ns + "valid_estimated_vel_diff_time", acc_param_.valid_est_vel_diff_time);
+    update_parameter(parameters, acc_ns + "valid_vel_que_time", acc_param_.valid_vel_que_time);
+    update_parameter(parameters, acc_ns + "valid_estimated_vel_max", acc_param_.valid_est_vel_max);
+    update_parameter(parameters, acc_ns + "valid_estimated_vel_min", acc_param_.valid_est_vel_min);
+    update_parameter(parameters, acc_ns + "thresh_vel_to_stop", acc_param_.thresh_vel_to_stop);
+    update_parameter(parameters, acc_ns + "use_rough_velocity_estimation", acc_param_.use_rough_est_vel);
+    update_parameter(parameters, acc_ns + "rough_velocity_rate", acc_param_.rough_velocity_rate);
+
+    if (planner_) {
```

Comment:
Why you need this check? Is this function called before initialization?
It's okay, but just out of curiosity.

Reply:
I added a check just in case, but I think it is unnecessary, so I will delete it.